nmma/em/model.py

```diff
@@ -369,7 +368,7 @@ def observation_angle_conversion(self, parameters):
     def generate_lightcurve(self, sample_times, parameters):
         if self.parameter_conversion:
             new_parameters = parameters.copy()
-            new_parameters, _ = self.parameter_conversion(new_parameters, [])
```

Comment:
isn't the parameter_conversion for MMA takes not only the parameters dict but also the parameter keys?

Reply:
For my setup self.parameter_conversion points to `MultimessengerConversion.convert_to_multimessenger_parameters` in joint/conversion.py line 468 and that function takes only the parameters as dict. i get an error when i leave the second argument there